gtsam/discrete/DecisionTreeFactor.h

```diff
@@ -179,6 +179,14 @@ namespace gtsam {
       return total;
     }
 
+    /**
+     * @brief Prune the decision tree of discrete variables.
```

Comment:
Yeah, but explain what that *means*

Reply:
Did this in https://github.com/borglab/gtsam/pull/1151 🙂